### ISSUE TITLE
Skeleton Client missing?

### ISSUE THREAD

**Issue by zerjillo** [MMTObservatory/pyINDI#13]
I cannot find the skeleton client file mentioned in the readme to create a client. Have I missed something? Is there any simple code to connect to a server, get some properties and update some values?
Thanks

**Comment by srswinde** [comment]
Indeed, we have been doing a lot of refactoring on the client side and have removed the skeleton client in favor of the [gui.py](https://github.com/MMTObservatory/pyINDI/blob/master/example_clients/gui/gui.py) script. If you install pyindi and change gui.py to reflect your indiserver host and port you should be able to view the web interface at http://localhost:5905. Let me know if this works for you. I will update the readme soon to explain the new way of running the client.
